apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/AgentClassLoader.java

```diff
@@ -72,89 +100,27 @@ public static AgentClassLoader getDefault() {
      *
      * @throws AgentPackageNotFoundException if agent package is not found.
      */
-    public static void initDefaultLoader() throws AgentPackageNotFoundException {
+    public static void initDefaultLoader() {
         if (DEFAULT_LOADER == null) {
             synchronized (AgentClassLoader.class) {
                 if (DEFAULT_LOADER == null) {
-                    DEFAULT_LOADER = new AgentClassLoader(PluginBootstrap.class.getClassLoader());
+                    DEFAULT_LOADER = new AgentClassLoader(PLUGIN_JAR_URLS, PluginBootstrap.class.getClassLoader());
                 }
             }
         }
     }
 
-    public AgentClassLoader(ClassLoader parent) throws AgentPackageNotFoundException {
-        super(parent);
```

Comment:
This super should already make the class loader reading target class(like Jedis class).

Reply:
Not so, because you override the `findClass` method, which only reads the plugin directory and does not call parent `findClass`

Reply:
`ClassLoader#loadClass` should cover similar logic about looking for parent class loader too.